Glyssen/ProjectExporter.cs

```diff
@@ -336,6 +338,8 @@ private void GenerateExcelFile(string path, IEnumerable<List<object>> data)
 				var sheet = xls.Workbook.Worksheets.Add("Script");
 				sheet.Cells["A1"].LoadFromArrays(dataArray);
 
+				ColorizeAnnotations(sheet.Cells);
```

Comment:
Probably can just limit to the primary (and secondary, if present) reference text columns. It's be slightly more efficient and maybe make the intention of the code a little clearer.